students/wielgus_daniel/lesson_01_basics/lesson_01_part_01.py

```diff
@@ -0,0 +1,78 @@
+import math
+
+#Problem1: Sum of numbers
+print("We will count the Sum of three numbers")
+print("Please write first number")
+firstNumber = int(input())
+print("Your choice is: " + str(firstNumber))
+print("Please write second number")
+secondNumber = int(input())
+print("Your choice is: " + str(secondNumber))
+print("Please write third number")
+thirdNumber = int(input())
+print("Your choice is: " + str(thirdNumber))
+print("Your choose numbers: " + str(firstNumber) + ", " + str(secondNumber) + ", " + str(thirdNumber))
+sum = firstNumber + secondNumber + thirdNumber
+print("The sum of yours numbers is: " + str(int(sum)))
```

Comment:
Variable name sum shadows python built-in function sum()
https://docs.python.org/3/library/functions.html
Please avoid that in future.